ansible-ipi-install/roles/installer/tasks/60_deploy_ocp.yml

```diff
@@ -7,4 +13,23 @@
 - name: Deploy OpenShift Cluster
   shell: |
     /usr/local/bin/openshift-baremetal-install --dir {{ dir }} --log-level debug create cluster
```

Comment:
Lets keep this exactly the same but add a when clause:
```
when: bootstrap_timeout is undefined and install_timeout is undefined
```

Reply:
So your suggestion is to deploy as normal without the async anytime the `increase` values are undefined? I would argue that the async and separate `wait-for` tasks have value on their own without the increased timeouts since they provide more incremental feedback to the user during the install process.